flamedisx/xenon/x1t_sr1.py

```diff
@@ -298,14 +309,27 @@ def random_truth(self, n_events, fix_truth=None, **params):
         d['z_observed'] = d['z']
 
         # Adding some smear according to posrec resolution
-        d['x_observed'] = np.random.normal(d['x_observed'].values, scale=0.4) # 4 mm resolution)
-        d['y_observed'] = np.random.normal(d['y_observed'].values, scale=0.4) # 4 mm resolution)
+        d['x_observed'] = np.random.normal(d['x_observed'].values, scale=4) # 0.4 = 4 mm resolution)
```

Comment:
@cecilia-ferrari Could you link to some internal xenon note that you got this number from(that position reconstruction uncertainty ~4cm)? From [Jaron's note](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenonnt:analysis:reconstruction_team:s1_xyz_rec_with_data_driven), it does seem to be on the order of 4cm, but [Matteo's note](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenonnt:analysis:reconstruction_team:s1_xyz_rec_with_data_driven) seems to suggest smt half the value. It's not a huge difference but would be good to document why we increased the posrec smearing by a factor of 10.

Reply:
Ah! you found a modification that I introduced to improve the fit with not so great results. I do think that at this point this smearing along with the field maps should undergo a review process in order to fix everything with one commit. Tomorrow me and @eangelino will have a call and discuss also about it. commits will be released soon to define once for all the correct implementation :muscle:

Reply:
What time's the call, I might be able to join.

Reply:
I'm asking Shixiao and the other MC guys about the recommended value to put as the posrec resolution. But since this is a simple change, we can update the value in a later PR.